src/net.cpp

```diff
@@ -417,6 +417,9 @@ CNode* CConnman::ConnectNode(CAddress addrConnect, const char *pszDest, bool fCo
     // Resolve
     const uint16_t default_port{pszDest != nullptr ? GetDefaultPort(pszDest) :
                                                      m_params.GetDefaultPort()};
+
+    // Collection of addresses to try to connect to: either all dns resolved addresses (if pszDest is provided) or addrConnect otherwise.
```

Comment:
nit: would be nice to say `XYZ (pszDest) is provided` instead of `pszDest is provided`